copy-project.rst

```diff
@@ -0,0 +1,27 @@
+.. _copy-project:
```

Comment:
use the form:

    .. _how-to-[...]:

In this case it should be:

    .. _how-to-project-duplicate: